### DISCUSSION TITLE
[Feature] Use currently shown photo as album cover

### DISCUSSION THREAD

**Discussion by waclaw66** [immich-app/immich#3449] [Feature Request]
### The feature

Add "Use as album cover" item to context menu (mobile doesn't have such) of currently shown photo.

Motivation comes from usual usecase. Viewing images in an album, like a photo as cover. Then I need to cancel viewing, select "Set album cover" from album context menu and find that photo again and select it. Too clumsy.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by hdagar** [comment]
This would be a **really** useful feature (on mobile app) and is a good qol improvement. One of the first things you do when setting up an album is to set the cover image to aid with easier location of photos in the future. Often you need to alter this selection, especially if you add more photos to that same album at a later point.

**Reply by waclaw66** [reply]
This has been already implemented on web. However still badly missing on mobile, where is no option to set album cover at all.

**Reply by hdagar** [reply]
thanks, I've updated my comment to reflect it needs focus on the mobile app.

**Comment by mjtf3** [comment]
this is already implemented on android (maybe close the discussion as completed?)

**Reply by bo0tzz** [reply]
Yeah seems like this has been in for a while, not sure why it didn't get closed, thanks!
